### PR TITLE
Telegram imported message pretty

### LINKED ISSUE #587: Make Telegram imported message pretty

### PR DESCRIPTION
**Pull request by BykhovDenis**
close #587 Signed-off-by: Denis Bykhov [bykhov.denis@gmail.com](mailto:bykhov.denis@gmail.com)

### CODE REVIEW

File: models/telegram/src/index.ts [hunk 1]
```diff
@@ -39,28 +41,63 @@ export class TTelegramMessage extends TDoc implements TelegramMessage {
   incoming!: boolean
 }
 
+@Model(telegram.class.SharedMessages, core.class.AttachedDoc, DOMAIN_TELEGRAM)
+export class TSharedTelegramMessages extends TAttachedDoc implements SharedTelegramMessages {
+  @Prop(TypeString(), 'Messages' as IntlString)
```
Comment (haiodo) [inline models/telegram/src/index.ts]: why not Collection?

File: plugins/telegram-assets/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "version": "0.6.0",
+  "main": "src/index.ts",
+  "author": "Anticrm Platform Contributors",
+  "license": "EPL-2.0",
+  "scripts": {
+    "build": "",
+    "build:docs": "api-extractor run --local",
+    "lint": "ts-standard src",
+    "lint:fix": "ts-standard --fix src",
+    "format": "prettier --write 'src/**/*.{ts*,js*,yml}' && ts-standard --fix src"
```
Comment (haiodo) [inline plugins/telegram-assets/package.json]: Let's add clean code for new code. There is no dependency to ts-standard or prettier in plugin.

**Comment by BykhovDenis** [comment]
изображение

File: models/telegram/src/index.ts (hunk 1 above)
Comment (BykhovDenis) [reply models/telegram/src/index.ts]: Good catch

**Review by aplatoff — APPROVED** [review]

**Pull request merged**